libs/pre/datamodel/preprocessorbcdataitem.h

```diff
@@ -82,9 +87,7 @@ public slots:
 	bool showDialog();
 
 private slots:
-	void setModified(bool modified = true) override {
-		PreProcessorDataItem::setModified(modified);
-	}
+	void setModified(bool modified = true);

```

Comment:
Hi Keisuke,

Shouldn't this still have override attached to it?

Thanks,
Scott